build.sh

```diff
@@ -178,15 +178,16 @@ if (( NUMARGS == 0 )) || hasArg rmm; then
     cd "${REPODIR}/python"
     export INSTALL_PREFIX
     echo "building rmm..."
+
     if [[ ${CUDA_MALLOC_ASYNC_SUPPORT} == OFF ]]; then
-        python setup.py build_ext_no_async --inplace
+        python setup.py build_ext --inplace -- -DCUDA_MALLOC_ASYNC_SUPPORT=OFF -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX}
     else
-        python setup.py build_ext --inplace
+        python setup.py build_ext --inplace -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX}
```

Comment:
I think this needs the `--` as well?
```suggestion
        python setup.py build_ext --inplace -- -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX}
```

Reply:
Thanks for catching that